src/config/bluetooth.md

```diff
@@ -0,0 +1,84 @@
+# Bluetooth
+
+Ensure the bluetooth controller is not blocked. Use `rfkill` to check whether
+there are any blocks and to remove soft blocks. If there is a hard block, there
+is likely either a physical hardware switch or an option in the BIOS to enable
+the bluetooth controller.
+
+```
+$ rfkill
+ID TYPE     DEVICE      SOFT      HARD
+0 wlan      phy0   unblocked unblocked
+1 bluetooth hci0     blocked unblocked
+
+# rfkill unblock bluetooth
+```
+
+## Installation
+
+Install the `bluez` package.
+
+```
+# xbps-install -S bluez
+```
+
+Enable the `bluetoothd` and `dbus` services.
+
+```
+# ln -s /etc/sv/bluetoothd /var/service
+# ln -s /etc/sv/dbus /var/service
+```
+
+Add your user to the `lp` group and restart the `dbus` service.
+
+```
+# usermod -a -G lp $USER
+# sv restart dbus
+```
+
+## Usage
+
+Manage bluetooth connections and controllers using `bluetoothctl`. It uses a
+command line interface; to find out what commands are available, enter `help`.
+To exit, enter `exit` or `quit`.
+
+```
+$ bluetoothctl
+
+[bluetooth]# help
+Menu main:
+Available commands:
+-------------------
+advertise             Advertise Options Submenu
+scan                  Scan Options Submenu
+gatt                  Generic Attribute Submenu
+list                  List available controllers
+show [ctrl]           Controller information
+select <ctrl>         Select default controller
+devices               List available devices
+paired-devices        List paired devices
+...
+```
+
+Consult the [Arch Wiki](https://wiki.archlinux.org/index.php/Bluetooth#Pairing)
+page for an example on how to pair a device.
+
+> Note: To use an audio device such as a wireless speaker or headset, you must
+> additionally use either the [PulseAudio](./media/pulseaudio.md) sound server
```

Comment:
"... ALSA users need to install the `bluez-alsa` package, while PusleAudio users do not need additional software."